sktime/forecasting/hf_momentfm_forecaster.py

```diff
@@ -0,0 +1,535 @@
+"""Interface for the momentfm deep learning time series forecaster."""
+
+import warnings
+
+import numpy as np
+import pandas as pd
+from skbase.utils.dependencies import _check_soft_dependencies
```

Comment:
according to @fkiraly I learnt today that it is currently better to use the sktime version instead of the skbase version

Reply:
switched it to sktime.utils